packages/corejs/test/examples/damage.example.ts

```diff
@@ -0,0 +1,6 @@
+const damageExample = {
+  damage_type: 'body_crack',
+  part_type: 'ignore'
+}
+
+export default damageExample;
```

Comment:
You can set keys in camelCase even in examples because we are mapping deep the keys to transform snake_case to camelCase. I'm not happy about it, but this is what we expected working with JS.